src/lang/builtins_maybe.ml

```diff
@@ -0,0 +1,52 @@
+(*****************************************************************************
+
+  Liquidsoap, a programmable audio stream generator.
+  Copyright 2003-2019 Savonet team
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details, fully stated in the COPYING
+  file at the root of the liquidsoap distribution.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+
+ *****************************************************************************)
+
+open Lang_builtins
+
+let () = Lang.add_module "maybe"
+
+let () =
+  add_builtin "maybe.nothing" ~cat:Liq ~descr:"Nothing." []
+    (Lang.maybe_t (Lang.univ_t ()))
+    (fun _ -> Lang.nothing)
```

Comment:
I am actually thinking that it could be a good idea to call it `null` as in other languages?
```suggestion
  add_builtin "maybe.null" ~cat:Liq ~descr:"Nothing." []
    (Lang.maybe_t (Lang.univ_t ()))
    (fun _ -> Lang.null)
```

Reply:
Yes!

Reply:
Why not at top level then?